packages/evm/src/precompiles/01-ecrecover.ts

```diff
@@ -13,13 +13,16 @@ import { OOGResult } from '../evm.js'
 
 import { gasLimitCheck } from './util.js'
 
+import { getPrecompileName } from './index.js'
+
 import type { ExecResult } from '../types.js'
 import type { PrecompileInput } from './types.js'
 
 export function precompile01(opts: PrecompileInput): ExecResult {
+  const pName = getPrecompileName('01')
```

Comment:
This constant can be moved outside this method, such that it does not get called each time if this precompile gets called (micro-optimization). (For all precompiles)

Reply:
Will do.

Reply:
(and totally not so "micro", makes a lot of sense)

Reply:
Ah, this does not seem to work, CI breaks on several occasions:

<img width="854" alt="grafik" src="https://github.com/user-attachments/assets/2e4e68e0-c663-49ae-914f-70a8baf7708f">

When looking at the code, I think this is due to circular imports (this whole precompile code is generally not so optimally organized, but I would not want to expand this into a full round refactor), so the `precompiles/index.ts` file is importing all the precompiles but is then itself *exporting* this `getPrecompileName` method, this does not seem to be able to be resolved.

Can we therefore please keep the non-optimized version? Will delete the respective commit again and re-push. I do not have an easy fix for this.